substrate/frame/utility/src/lib.rs

```diff
@@ -93,18 +97,23 @@ pub mod pallet {
 			+ IsSubType<Call<Self>>
 			+ IsType<<Self as frame_system::Config>::RuntimeCall>;
 
-		/// The caller origin, overarching type of all pallets origins.
-		type PalletsOrigin: Parameter +
-			Into<<Self as frame_system::Config>::RuntimeOrigin> +
-			IsType<<<Self as frame_system::Config>::RuntimeOrigin as frame_support::traits::OriginTrait>::PalletsOrigin>;
+		/// The type to convert a pallet's origin to it's sovereign account ID used to dispatch a
```

Comment:
Sovereign account associated with a pallet origin feels like wrong phrasing, but I'm not sure.

```suggestion
		/// The type to convert a pallet's origin to an account ID used to dispatch a
```